torch/nn/intrinsic/quantized/modules/__init__.py

```diff
@@ -1,10 +1,10 @@
-
-from .linear_relu import LinearReLU
+from .linear_relu import LinearReLU, LinearReLUBackendIndependent
 from .conv_relu import ConvReLU1d, ConvReLU2d, ConvReLU3d
 from .bn_relu import BNReLU2d, BNReLU3d
 
 __all__ = [
     'LinearReLU',
+    'LinearReLUBackendIndependent',
```

Comment:
NIT: Should we create a namespace nn.quantized.reference and have Linear, LinearRelu in there? The name BackendIndependent could be confusing to external users. Also, if we dont intend this for direct use, should we prefix with an _?, i.e have:
nn.quantized._reference.LinearRelu

Reply:
Sure

Reply:
@raghuramank100 do you also suggest to change backend_independent to reference in the api?

Reply:
changed to reference